### DISCUSSION TITLE
What's the difference with readyset?

### DISCUSSION THREAD

**Discussion by xudong963** [getdozer/dozer#651] [General]
I've had a quick look at dozer, and it seems similar to https://readyset.io/?

**Comment by snork-alt** [comment]
Hi @xudong963,

ReadySet is purely a caching layer for MySQL and PostgreSQL. Dozer is fundamentally a platform to allow users to efficiently expose data through APIs. We interface with both relational databases as well as Data Warehouses such as Snowflake.

We allow the user to determine what is the best way to cache and expose their data. Thus, we directly expose gRPC and REST APIs so that data can be integrated very easily with other microservices or frontends. We see caching as one of the components needed to efficiently build data APIs. However, there are many other aspects that we take care of such as different types of indexing, real-time updates, schema validation, and evolution, etc.

**Reply by xudong963** [reply]
Thanks @snork-alt
